dotcom-rendering/src/web/components/CommentBlockComponent.stories.tsx

```diff
@@ -26,3 +26,40 @@ export const defaultStory = () => {
 	);
 };
 defaultStory.story = { name: 'default' };
+
+export const embeddedBlockquoteStory = () => {
+	const bodyContent = `
+		<blockquote>
+			<p>
+				Children and youth are striking to demand action. But many of not most are
+				themselves involved in one of the most damaging practices towards earth's
+				climate, I mean consuming massive amounts of meat and diary in their diets.
+			</p>
+			<p>What is their thought on this regard ?</p>
+		</blockquote>
+		<p>
+			Are they? I see young people leading on this. There's been a massive surge
+			towards veganism by young people, which has resulted in a sevenfold increase
+			in the UK within 5 years. Quite remarkable. And it should be recognised.
+		</p>
+	`;
+	return (
+		<div
+			css={css`
+				padding: 20px;
```

Comment:
```suggestion
				padding: 20px;
				width: 700px;
```

Reply:
Nitpicking  but it's nice if the story has a bit of context for how the content might look